etl-job/fhir_import_export.py

```diff
@@ -357,21 +314,17 @@ def _empty_project(output: list[str],
     """Clear out graph and flat metadata for project """
     # check permissions
     try:
-        empty_project(program=program, project=project, dictionary_path=dictionary_path, config_path=config_path)
+        grip_delete(graph_name="CALIPER", project_id=f"{program}-{project}",
```

Comment:
Should grip's URL and PORT be passed as well?

Reply:
Seems the pod has the best information (it's env) to determine grip url, port etc.  
Should we pass that information to the grip submission layer?

Reply:
Yes thinking about this further, need to add this.

Reply:
see https://github.com/ACED-IDP/gen3-helm/pull/67